pysc2/lib/features.py

```diff
@@ -344,6 +370,47 @@ def unit_vec(u):
         out["build_queue"] = np.stack(unit_vec(u)
                                       for u in ui.production.build_queue)
 
+    def raw_unit_vec(u):
+      screen_pos = self._world_to_fl_screen.fwd_pt(point.Point.build(u.pos))
+      screen_radius = self._world_to_fl_screen.fwd_dist(u.radius)
+      return np.array((
```

Comment:
Could the first of these match the order of unit_vec?

Reply:
Done